keyGenerartor/src/base/KeyGenerator.java

```diff
@@ -4,12 +4,12 @@
 
 public class KeyGenerator {
 
-	public static String[] Generator(Integer keyLenght, String currentDictionary) {
-		String[] key = new String[keyLenght];
+	public static String Generator(Integer keyLenght, String currentDictionary) {
+		String key = null;
```

Comment:
Try to run with this change:
```suggestion
		String key = "";
```